packages/components/panel/src/panel.ts

```diff
@@ -42,34 +55,75 @@ export class Panel extends ScopedElementsMixin(LitElement) {
   /** @internal */
   static override styles: CSSResultGroup = styles;
 
+  /** Indicates whether the toggle button has been clicked. */
+  #toggleClicked = false;
```

Comment:
I'm not sure why this is necessary. You use it for the icon, but you could also just do `:host([expanded]) sl-icon { ... }`?

Reply:
Right, I'll change it. This is just a relict from the previous version of the animation (previously we wanted to make an animation with chevron and dash).